Release bigtable 0.30.0

sduskis opened the pull request:
This pull request was generated using releasetool.

Closes #5770.

theacodes commented on bigtable/CHANGELOG.md:
```diff
@@ -4,6 +4,76 @@
+-- BigTable: Add admin app profile methods on Instance (#5315)
+-- BigTable: Add data app profile id  (#5369)
+
+### Dependencies
```
you can drop empty sections

tseaver commented:
@sduskis Do we want to land #5838 before making a release?

sduskis commented:
@tseaver, we're going to release a Beta release in the near future, and I'm ok deferring #5838 in order to get this release out.

tseaver commented on bigtable/CHANGELOG.md:
```diff
@@ -4,6 +4,76 @@
+- Add Test runs for Python 3.7 and remove 3.4 (#5295)
+- disable bigtable system tests (#5381)
+- Modify system tests to use prerelease versions of grpcio (#5304)
+- Override gRPC max message lengths. (#5498)
```
For release notes, please normalize as follows:

- Strip any `Bigtable: ` prefixes
- Start each bullet point with a verb in imperatlve voice ("Improve foo" rather than "Improved foo" or "Improving foo").
- Link the relevant PR (most have this, but some are missing).

sduskis commented:
@tseaver, I think I addressed most of your concerns.  I was not able to get PRs for some line items that were committed directly on master without a PR.

tseaver commented:
@sduskis I've cleaned up a bit, fixing the nesting, putting the bullet points into imperative mode, and moving all the test-only changes to the bottom section.  PTAL.

tseaver commented:
@sduskis Looks like you just overwrote my changes. :(

sduskis commented:
@tseaver, I reverted to your version...  Sorry about that.

tseaver commented:
Release is made: https://github.com/GoogleCloudPlatform/google-cloud-python/releases/tag/bigtable-0.30.0

tseaver commented:
Tag build succeeded, [pushed distributions to PyPI](https://pypi.org/project/google-cloud-bigtable/0.30.0/).
